geode-docs/developing/region_options/dynamic_region_creation.html.md.erb

```diff
@@ -21,15 +21,20 @@ limitations under the License.
 
 You can dynamically create regions in your application code and automatically instantiate them on members of a cluster.
 
-If your application does not require partitioned regions, you can use the <span class="keyword apiname">org.apache.geode.cache.DynamicRegionFactory</span> class to dynamically create regions, or you can create them using the `<dynamic-region-factory>` element in the cache.xml file that defines the region. See [&lt;dynamic-region-factory&gt;](../../reference/topics/cache_xml.html#dynamic-region-factory).
+If your application does not require partitioned regions, you can use the `org.apache.geode.cache.DynamicRegionFactory` class to dynamically create regions, or
```

Comment:
DynamicRegionFactory has been deprecated in favor of using the function service instead. The javadocs on the class have this:  * @deprecated This class is deprecated. Use {@link FunctionService} to create regions on other
 *             members instead.
So I would use this ticket as an opportunity to remove this reference to DynamicRegionFactory and just keep the part about using function execution to create a region dynamically.
Note that both the class DynamicRegionFactory and the corresponding xml element <dynamic-region-factory> are deprecated so they could be removed in the next major release

Reply:
Thanks, @dschneider-pivotal.
As a courtesy to developers of legacy apps that use the DynamicRegionFactory class, I'm leaving it in place for this iteration, but noting that it's deprecated and demoting it from first position on the page. Future deletion is definitely in order.

Reply:
I'd say either direction is ok. Go ahead note that it's deprecated but keep it in place. Maybe the correct time to delete the documentation is in the release that actually deletes the DynamicRegionFactory class.